src/thor/costmatrix.cc

```diff
@@ -239,10 +217,10 @@ void CostMatrix::SourceToTarget(Api& request,
   for (const auto& connection : best_connection_) {
     uint32_t target_idx = count % target_location_list.size();
     uint32_t origin_idx = count / target_location_list.size();
-    float time = connection.cost.secs + .5f;
+    float time = connection.cost.secs;
```

Comment:
this was another bug, we shouldn't add 0.5 at this point.. also reflected in the tests

Reply:
yeah why would that be there? does it have something to do with trivial matrices and avoid a 0 seconds?

Reply:
no, it's there because I messed up on a previous PR and it wasn't caught in the review. I can only guess that I wanted to round, but it's for sure the wrong place for that:)